Kitodo/src/main/webapp/pages/processes.xhtml

```diff
@@ -22,7 +22,8 @@
             <h3>#{msgs.prozessverwaltung}</h3>
             <h:form id="addForm">
                 <p:menuButton value="#{msgs.newElement}" icon="fa fa-plus-circle fa-lg" iconPos="right" styleClass="button-filled-green">
-                    <p:menuitem value="#{msgs.newProcess}" outcome="/pages/processEdit" icon="ui-icon-plusthick" />
+                    <!-- TODO: decide whether creating processes from scratch - without template - is required (than we need the next line) -->
```

Comment:
In description of this pull request you are writing that it should not possible any more to create processes without a process template. I read this as this lines can be removed.

Reply:
No, the current implementation is not final:

> the "New process" sub-button under the "New" button on the processes page and the "is template" switch on the "Create new process (template)" page have been removed **until we decide whether this way to create new processes without templates is really required or not**